### ISSUE TITLE
Feature Request: Running shaders as skybox

### ISSUE THREAD

**Issue by wallcarpet40** [galister/wlx-overlay-s#104]
I saw a KDE Plasma plugin that runs shaders as desktop wallpaper and I was wondering, if something like that could be used as a skybox for Wlx-overlay?

https://github.com/y4my4my4m/kde-shader-wallpaper/?tab=readme-ov-file#videos

**Comment by galister** [comment]
Of course! However, shaders are compiled at build time, so as of now, the shader would have to be patched in there.
